### ISSUE TITLE
[HELP] How-to make Jailbreak iPhone5

### ISSUE THREAD

**Issue by dalisoft** [zhengmin1989/iOS_ICE_AND_FIRE#4]
Hi @zhengmin1989 
First of all, thanks for amazing Jailbreak tool.
How i can do Jailbreak iphone 5 iOS 9.2...
i have Windows 7, iphone 5, internet connection, and some web skills.
there have some video tutorial or other way? please share with us.
Thanks.

**Comment by heefan** [comment]
iOS9.2 cannot be jail-broken as I know. Only for iOS9.1
check it out here [WIKI](https://en.wikipedia.org/wiki/IOS_jailbreaking)

**Comment by dalisoft** [comment]
Hi @heefan. Thanks for reply. I can upgrade to iOS 9.2.1. then how to compile this app?

**Comment by heefan** [comment]
@dalisoft 
It doesn't make sense to upgrade to iOS9.2.1. Jail-breaking tools only target for iOS9.0 and iOS 9.1
(please read WIKI, it mentions about that)

Plus, you cannot install iOS 9.2.1 from Apple now, only iOS 9.3.2 is available. 
check out this website :  https://ipsw.me/
Select your phone model and you will see that. 

Cheers

**Comment by dalisoft** [comment]
Then, this app isn't real? In twitter written that this app can be jailbreak iDevice as iPhone 5like 32bit arch. So i retweet and write it is not work

**Comment by heefan** [comment]
@dalisoft,  do you mean iDevice from the [webpage](http://www.wikihow.com/Jailbreak-Your-iDevice-Right-from-It)?

**Comment by cellinho** [comment]
@dalisoft dont do anything mate... wasting time

**Comment by heefan** [comment]
Yep, shall I know what's your requirement for jail-breaking? For example, 

```
Device: iPhone 5
OS:  iOS9.2
```

**Intent**
I want to jail-break iPhone5.

**Motivation**
I would like to know safe way to process it. 

Probably I can provide more information to you.

**Comment by dalisoft** [comment]
@heefan 
Here what i want: https://github.com/zhengmin1989/iOS_ICE_AND_FIRE/issues/5
So, i know what requirements:
Device: iPhone (32-bit ARCH only)
OS: iOS <9.3, best for iOS 9.2.x
Manf: Apple.

What else you need to give me instruction

**Comment by heefan** [comment]
@dalisoft , Don't panic dude

```
1. please backup first.
2. please note, you cannot resume back to lower iOS if you upgrade to iOS9.3.2
```

For 2, see the screenshot as following, 
![screen shot 2016-06-16 at 10 19 36 am](https://cloud.githubusercontent.com/assets/892641/16103827/7deddad6-33ae-11e6-9254-b381529c44a1.png)
only iOS9.3.2 is available from Apple Server. 

---

For `iOS9.1`

Let rock it, 
1. Download  [Pangu](http://en.pangu.io/) for `iOS9.1` jail-broken tool.
2. Follow the video tutorial from youtube. [How to jailbreak iOS 9 with Pangu](https://www.youtube.com/watch?v=6QLlTXWfUak)

It is easy, right?

If you tell me, how to jial-break iOS9.2, forget about it dude, I cannot help you.

**Comment by dalisoft** [comment]
@heefan but i need another. i told you i'm in iOS 9.2 and Pangu hasn't released yet for iOS 9.2
